src/main.cpp

```diff
@@ -1375,12 +1375,34 @@ class ATTRIBUTE_HIDDEN FragmentedSampleReader : public SampleReader, public AP4_
   };
   uint32_t GetTimeScale() const override { return m_track->GetMediaTimeScale(); };
 
+public:
+  static const AP4_UI32 TRACKID_UNKNOWN = -1;
+
```

Comment:
I've added the `public` again as I can see this convention further down for the private member variables in this class (only one public function above but it's several pages long). This should aid readability but my question is, is this right or not?

Reply:
i think that for now is ok
with the refactor related code i think will be moved under another file
then this variable can be moved in the header file as `constexpr`